WalletWasabi.Fluent/Controls/LineChart.cs

```diff
@@ -384,7 +384,8 @@ private double GetCursorPosition(double areaWidth)
 			{
 				if (xAxisValues[i] <= xAxisCurrentValue)
 				{
-					return areaWidth / xAxisValues.Count * i;
+					var cursorPosition = i == 0 ? 0.0 : areaWidth / (xAxisValues.Count - 1) * i;
```

Comment:
Is it possible to have a `DivideByZeroException` here?

Reply:
It's about having correct value when Count == 1, otherwise we will get double.NaN

Reply:
@wieslawsoltes what happens if `xAxisValues.Count == 1` and `i == 1`... ?

Reply:
Cannot happen, if count == 1 then i would be 0 and after the cycle, it will exit from the for loop.